db/queries/scores/queries.py

```diff
@@ -0,0 +1,63 @@
+"""Queries which are performed on the `scores` table.
+
+Joins allowed.
+"""
+import json
+from typing import Dict
+from typing import List
+
+from db import db
+from db.models.score import JustScore
+from db.queries.assessment_records._helpers import derive_values_from_json
+from db.schemas import JustScoreMetadata
+from sqlalchemy import func
+from sqlalchemy import select
+from sqlalchemy.orm import defer
+
+
+def get_just_score_for_application_sub_crit(
```

Comment:
⛏️ This needs to be more accurate - it selects the most recent score.

Reply:
done!